listenbrainz/db/recommendations_cf_recording_feedback.py

```diff
@@ -105,3 +106,32 @@ def get_feedback_count_for_user(user_id: int) -> int:
         count = int(result.fetchone()["count"])
 
     return count
+
+
+def get_feedback_for_multiple_recordings_for_user(user_id: int, recording_list: List[str]):
+    """ Get a list of recording feedback given by the user for given recordings
+
+        Args:
+            user_id: the row ID of the user in the DB
+            recording_list: list of recording_mbid for which feedback records are to be obtained
+                            - if record is present then return it
+                            - if record is not present then return rating = None
+
+        Returns:
+            A list of Feedback objects
+    """
+
+    args = {"user_id": user_id, "recording_list": tuple(recording_list)}
```

Comment:
@mayhem okay so the query errored out because I didn't convert the list to tuple. phew

Reply:
I had suspected as much -- this can be quite tricky and has caught me off guard a number of times.

Reply:
Looks much much better!